python/rpdk/python/codegen.py

```diff
@@ -221,6 +226,10 @@ def _make_pip_command(base_path):
             str(base_path / "build"),
         ]
 
+    @staticmethod
+    def _get_plugin_information():
+        return {"plugin-tool-version": __version__, "plugin-name": "python"}
```

Comment:
How we get the plugin runtime version?  which is more important than the tool version.